shared/utils/count_oval_objects.py

```diff
@@ -111,7 +111,19 @@ def main():
         rule_id = rule.attrib['id']
         oval_refs = []
         for ref in rule.findall(".//check-content-ref"):
-            oval_name = ref.attrib['name']
+
+            # Skip remotely referenced OVAL checks since they won't have the
+            # 'name' attribute set (just 'href' would be set in that case)
+            try:
+                oval_name = ref.attrib['name']
+            except KeyError:
+                if ref.attrib['href'] is not None:
```

Comment:
The ```attrib``` method will return ```None``` if the attribute doesn't exist:
&nbsp; &nbsp; [1]  http://lxml.de/api/xml.etree.ElementTree.Element-class.html

Have a look at ```attrib``` source code.

So IMHO this should be safe (but can test that).